src/main/java/appeng/me/storage/CellInventoryHandler.java

```diff
@@ -137,7 +138,7 @@ public IncludeExclude getIncludeExcludeMode()
 		return this.getWhitelist();
 	}
 
-	NBTTagCompound openNbtData()
+	private NBTTagCompound openNbtData()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private method 'openNbtData' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedPrivateMethod)